### PR TITLE
Dockerfile: update Go to 1.13

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

**Comment by paulfantom** [comment]
Remember to also change it in https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-monitoring-operator/openshift-cluster-monitoring-operator-master.yaml and files responsible for 4.4 and 4.5 releases

**Comment by simonpasquier** [comment]
> Remember to also change it in https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-monitoring-operator/openshift-cluster-monitoring-operator-master.yaml and files responsible for 4.4 and 4.5 releases

Thanks, I completely forgot about this. So the `FROM` directive from the Dockerfile doesn't do anything because the downstream build overrides it too. I'm closing this PR since `-mod=vendor` is already enforced and I'll open another against openshift/release...

**Comment by paulfantom** [comment]
> So the FROM directive from the Dockerfile doesn't do anything because the downstream build overrides it too.

Yes, exactly.

**Comment by simonpasquier** [comment]
/hold

waiting for #597

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,6 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
+# When bumping the Go version, don't forget to update the configuration of the
+# CI jobs in openshift/release.
```
Comment (s-urbaniak) [inline Dockerfile]: :+1: maybe a link too?

**Pull request merged**